Directory.Build.targets

```diff
@@ -22,6 +22,15 @@
   </PropertyGroup>
 
   <!-- Packaging -->
+  <ItemGroup Condition="'$(IsPackable)' == 'true'">
+    <None Include="$(LicenseFile)"
+          PackagePath="$([System.IO.Path]::GetFileName('$(LicenseFile)'))"
```

Comment:
If you leave `PackagePath` empty it puts it in the root right?

Reply:
`PackagePath` needs the file path supplied otherwise the custom packaging system throws:

```
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.build.tasks.packaging\6.0.0-beta.20621.6\build\Packaging.targets(1203,5): error : System.Exception: Target path 'C:\git\runtime3\' contains invalid characters. [C:\git\runtime3\src\libraries\Microsoft.Bcl.AsyncInterfaces\pkg\Microsoft.Bcl.AsyncInterfaces.pkgproj]
```